core/storage/email/gae_models.py

```diff
@@ -485,6 +485,20 @@ def _generate_unique_reply_to_id(cls):
 
         raise Exception('Unique id generator is producing too many collisions.')
 
+    @classmethod
+    def get_by_thread_id(cls, thread_id):
+        """Fetches the GeneralFeedbackEmailReplyToIdModel instance corresponding
+        to the given thread id.
+
+        Args:
+            thread_id: str. The ID of the thread.
+
+        Returns:
+            list(FeedbackEmailReplyToIdModel). A list of instance corresponding
+            to the given thread_id.
+        """
+        return cls.query(cls.thread_id == thread_id).fetch()
```

Comment:
Could we do this in the job, instead? Prefer not changing the public API an existing model for a temporary job.

Reply:
Done!